app/commands/lightning_talks/book.rb

```diff
@@ -0,0 +1,76 @@
+module Commands
+  module LightningTalks
+    class Book < Base
+      include Import[
+        book_lightning_repo: 'repositories.book_lightning_repo',
+        schedule_lightning_repo: 'repositories.schedule_lightning_repo',
+        lightning_talk_service: 'services.lightning_talk'
+      ]
+
+      private
+
+      def handle_call(message)
+        username = username_for(message.from)
+
+        send_message(
+          chat_id: message.chat.id,
+          text: '🧐 *Choose one of the available slots below*',
+          parse_mode: :markdown,
+          reply_markup: inline_keyboard(*available_slot_buttons(username))
+        )
+      end
+
+      def handle_callback(callback, args)
+        id = args.fetch('id') { raise(FallbackError) }
+        slot = schedule_lightning_repo.by_id(id) || raise(FallbackError)
+        username = username_for(callback.from)
+        confirmed = args.fetch('confirm', false)
```

Comment:
```
confirmed = args['confirm']

if confirmed.nil?
 ...
```

Reply:
I  wanted to separate getting command's context from logics and keep the code in the `if` part at a single level of abstraction

Reply:
kk